### DISCUSSION TITLE
How to make errors more descriptive?

### DISCUSSION THREAD

**Discussion by Marchiuzzz** [nuxt/nuxt#10478] [General]
Is it just my setup or is the error page completely useless? I get an error but no information about on which component this error occurred. I also constantly get "missing stack frames"
![image](https://user-images.githubusercontent.com/73941456/171397627-a6f70f9d-db7c-472e-9375-95c89499a5ad.png)

How can I find out where this crashed?

**Comment by Marchiuzzz** [comment]
Still waiting for a response on this :/

**Comment by ndragun92** [comment]
try to find in code where u use `transparent_bg`. It feels like it is an object which is undefined sometimes and while you try to access that property it will throw an error which means you will need to add additional validations
